lang/python/python-requests/patches/0001-Allow-urllib3-125x.patch

```diff
@@ -0,0 +1,33 @@
+Commit in requests repository - urllib1.25 branch: https://github.com/kennethreitz/requests/commit/d6b5b401e8d6141bcefa4a70ff1c836aa085120b
+Pull request: https://github.com/kennethreitz/requests/pull/5063
+
+diff --git a/requests/__init__.py b/requests/__init__.py
+index bc168ee..9a899df 100644
+--- a/requests/__init__.py
++++ b/requests/__init__.py
+@@ -57,10 +57,10 @@ def check_compatibility(urllib3_version, chardet_version):
+     # Check urllib3 for compatibility.
+     major, minor, patch = urllib3_version  # noqa: F811
+     major, minor, patch = int(major), int(minor), int(patch)
+-    # urllib3 >= 1.21.1, <= 1.24
++    # urllib3 >= 1.21.1, <= 1.25
+     assert major == 1
+     assert minor >= 21
+-    assert minor <= 24
++    assert minor <= 25
+
+     # Check chardet for compatibility.
+     major, minor, patch = chardet_version.split('.')[:3]
+diff --git a/setup.py b/setup.py
+index 10ce2c6..0d5d0cc 100755
+--- a/setup.py
++++ b/setup.py
+@@ -44,7 +44,7 @@ packages = ['requests']
+ requires = [
+     'chardet>=3.0.2,<3.1.0',
+     'idna>=2.5,<2.9',
+-    'urllib3>=1.21.1,<1.25',
++    'urllib3>=1.21.1,<1.26,!=1.25.0',
```

Comment:
curios: what does `!=1.25.0` do ?
i would have assumed that `'urllib3>=1.21.1,<1.26'` is sufficient here

Reply:
This one is what should be in a new version of requests. https://github.com/kennethreitz/requests/commit/d6b5b401e8d6141bcefa4a70ff1c836aa085120b I don't think that we want to have something different than it should be there. 

 Both things do the same. Except this new one prevents to use 1.25.0 version as [recommended](https://github.com/kennethreitz/requests/issues/5067#issuecomment-487573608) by the developer of urllib3.

Reply:
thanks for the info